src/main/resources/assets/js/application/recruiter/directives/tdprTableDirective.js

```diff
@@ -99,7 +106,7 @@ define(['angular', 'application/recruiter/tdprRecruiterModule'], function (angul
                     if (newValue !== undefined) {
                         if (newValue === null) {
                             scope.startWeekDay = previousStartWeekDay;
-                            daysLength = 7;
+                            daysLength = _days.length;
```

Comment:
This is used for feature for selecting day and then only one day view. Few lines below there is selecting only one day. It can be removed if we won't need it.